content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+# VM Template Name
+vm_name                 = "palette-edge-template"
+# VM Settings
+vm_guest_os_type        = "ubuntu64Guest"
+vm_version              = 14
+vm_firmware             = "bios"
+vm_cdrom_type           = "sata"
+vm_cpu_sockets          = 4
+vm_cpu_cores            = 1
+vm_mem_size             = 8192
+vm_disk_size            = 51200
+thin_provision          = true
+disk_eagerly_scrub      = false
+vm_disk_controller_type = ["pvscsi"]
+vm_network_card         = "vmxnet3"
+vm_boot_wait            = "5s"
+# ISO Objects
+iso                 = "build/palette-edge-installer.iso"
+iso_checksum        = "build/palette-edge-installer.iso.sha256"
+```
+
+
+<InfoBox>
+
+The **vsphere.hcl** file customization is out of the scope of this tutorial. To change these configurations, you must clone the [tutorials](https://github.com/spectrocloud/tutorials.git) repository and edit the **edge-native/vmware/packer/vsphere.hcl** file, rebuild the image using these [local build steps](https://github.com/spectrocloud/tutorials/blob/main/docs/docker.md#local-builds) and use that image in the previous `docker run` command.  
+
+</InfoBox>
+<br />
+
+
+## Provision VMs
+Once the VM template is ready, you can clone it to provision VMs. Note that these VMs will act as Edge hosts during cluster deployment. The next steps will use the [GOVC](https://github.com/vmware/govmomi/tree/main/govc#govc) tool to clone the VM template and provision VMs. 
+
+
+GOVC will also require the same VMWare vCenter details as environment variables, as you defined earlier in the **.packerenv** file. Use the following command to source the **.packerenv** file and echo one of the variables to ensure the variables are accessible on your host machine. 
+<br />
+
+```bash
+source .packerenv
+echo $PKR_VAR_vcenter_server
+```
+
+Use the following heredoc script to create a file, **.goenv**, containing the VMWare vCenter details as environment variables. The script will reuse the values sourced from the **.packerenv** file.
+<br />
+
+```bash
+cat << EOF > .goenv
+vcenter_server=$PKR_VAR_vcenter_server
+vcenter_username=$PKR_VAR_vcenter_username
+vcenter_password=$PKR_VAR_vcenter_password
+vcenter_datacenter=$PKR_VAR_vcenter_datacenter
+vcenter_datastore=$PKR_VAR_vcenter_datastore
+vcenter_resource_pool=$PKR_VAR_vcenter_resource_pool
+vcenter_folder=$PKR_VAR_vcenter_folder
+vcenter_cluster=$PKR_VAR_vcenter_cluster
+vcenter_network=$PKR_VAR_vcenter_network
+EOF
+```
+
+
+Issue the following command to clone the VM template and provision three VMs in the VMWare vCenter. The `--env-file` option in the command below will read the environment file. Then it changes to the **edge-native/vmware/clone_vm_template/** directory in the container and executes the **deploy-edge-host.sh** script to provision three VMs. 
+<br />
+
+```bash
+docker run -it --rm --env-file .goenv ghcr.io/spectrocloud/tutorials:1.0.6 sh --cpu-shares  "cd edge-native/vmware/clone_vm_template/ && ./deploy-edge-host.sh"
```

Comment:
Same feedback as earlier, break up the command to two or three lines. Same feedback about `--cpu-shares`